.github/workflows/wheels.yml

```diff
@@ -132,16 +132,16 @@ jobs:
         #os: [macos-10.15, windows-latest]
         #os: [macos-10.15, macOS-M1]
         os: [macos-10.15]
-        python_version: ["2.7", "3.6", "3.7", "3.8", "3.9", "3.10", "pypy-3.7-v7.3.3", "pypy-3.8-v7.3.7"]
+        python_version: ["2.7", "3.6", "3.7", "3.8", "3.9", "3.10", "pypy-3.7-v7.3.9", "pypy-3.8-v7.3.9", "pypy-3.9-v7.3.9"]
```

Comment:
It's better to always use the oldest PyPy version that works for the build. Their C-API emulation is more likely to be backward than forward compatible. It's probably ok to start with 7.3.9 when adding a new PyPy3.9, but let's stick with the old versions for 3.7 and 3.8.

Reply:
Understood.  Thx.